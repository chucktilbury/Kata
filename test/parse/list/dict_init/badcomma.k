;-------------------------------------------------------------------------------
; bad comma 
;-------------------------------------------------------------------------------

var dict name = ['asd':123 x 'asd':123]