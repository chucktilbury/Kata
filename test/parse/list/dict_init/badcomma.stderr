Syntax: /home/chuck/Src/Kata/test/parse/list/dict_init/badcomma.k: 5: 29: expected a ',' or a ']' but got symbol
Syntax: /home/chuck/Src/Kata/test/parse/list/dict_init/badcomma.k: 5: 35: expected an assignment item but got [
Syntax: /home/chuck/Src/Kata/test/parse/list/dict_init/badcomma.k: 5: 35: expected module item or end of file but got var
Syntax: /home/chuck/Src/Kata/test/parse/list/dict_init/badcomma.k: 5: 35: module cannot be empty
